tests/data/b008m7xk.ts

```diff
@@ -0,0 +1,115 @@
+import { IPlayerMetadataResponse } from 'src/types/responses/IPlayerMetadataResponse';
+
+export default {
```

Comment:
Couldn't find a nice way to import the JSON directly, so just converted the JSON to modules for importing.